app/src/main/java/com/example/testapp/data/LoginRequest.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.testapp.data
+
+
+data class LoginRequest(
```

Comment:
Please add Java documentation at class level and function level with parameter description